test/hashes-test.ts

```diff
@@ -0,0 +1,154 @@
+import assert from "assert";
```

Comment:
probably a merge issue, but this was renamed to just `hashes` and I think it's in the `test/utils` folder.